src/git.sh

```
@@ -1,42 +1,393 @@
+  else
+    echo "${changelog}"
   fi
+}
 
-  if [ "$1" == "--store" ]; then
-    echo "last_tag=${last_tag}"
-    echo "last_tag=${last_tag}" >> $GITHUB_OUTPUT
+# Mock command execution for testing
+_mock_command() {
+  if [ "${GIT_MOCK_COMMANDS:-false}" = "true" ]; then
+    echo "MOCK: $@" >> "${GIT_MOCK_OUTPUT:-/dev/null}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)